contrib_bots/bots/weather/readme.md

```diff
@@ -0,0 +1,13 @@
+# DefineBot
+
+* This is a bot that displays weather information to selected stream
+
+* Weather information is brought to the website using an OpenWeatherMap API. The bot posts the
+weather information to the stream from which the user inputs the message.
+If the user inputs city that does not exist the bot displays "Sorry, city not found" message.
+
+* Before the first use you have to generate the key to OpenWeatherMap api
+ and replace dummy in .weather_config file inside bot main directory
+
+![Example Usage](assets/screen1.png)
+![Wrong City](assets/screen2.png)
```

Comment:
Since you're not including `assets/screenshot.png` here, can you delete that file?